src/quacc/recipes/espresso/core.py

```diff
@@ -0,0 +1,211 @@
+    copy_files: list[str] | None = None,
+    **kwargs,
+) -> RunSchema:
+    """
+    Function to carry out a single-point calculation.
+
+    Parameters
+    ----------
+    atoms
+        Atoms object
+    **kwargs
+        Custom kwargs for the espresso calculator. Set a value to
+        `None` to remove a pre-existing key entirely. For a list of available
+        keys, refer to the `ase.calculators.espresso.Espresso` calculator.
+
+        !!! Info "Calculator defaults"
+
+            ```python
+            {
+                "ecutwfc": 40,
+                "ecutrho": 160,
+            }
+            ```
+
+    Returns
+    -------
+    RunSchema
+        Dictionary of results from [quacc.schemas.ase.summarize_run][]
+    """
+
+    calc_defaults = {
+        'input_data': {
+            'control': {
+                'calculation': 'scf',
+                'restart_mode': 'from_scratch',
+                'pseudo_dir': SETTINGS.ESPRESSO_PP_PATH,
+            },
+            'system': {
+                "ecutwfc": 60,
+                "ecutrho": 240,
+            },
+            'electrons': {
+                'conv_thr': 1e-8,
+                'mixing_mode': 'plain',
+                'mixing_beta': 0.7
+            }
+        }
+    }
+    
+    template = EspressoTemplate('pw')
+
+    return _base_job(
+        atoms,
+        preset=preset,
+        template=template,
+        calc_defaults=calc_defaults,
+        calc_swaps=kwargs,
+        additional_fields={"name": "pw.x static"},
+        copy_files=copy_files,
+    )
+
+@job
+def ph_job(
+    preset: str | None = None,
+    copy_files: list[str] | None = None,
+    **kwargs,
+) -> RunSchema:
+    """
+    Function to carry out a single-point calculation.
+
+    Parameters
+    ----------
+    atoms
+        Atoms object
+    **kwargs
+        Custom kwargs for the espresso calculator. Set a value to
+        `None` to remove a pre-existing key entirely. For a list of available
+        keys, refer to the `ase.calculators.espresso.Espresso` calculator.
+
+        !!! Info "Calculator defaults"
+
+            ```python
+            {
+                "ecutwfc": 40,
+                "ecutrho": 160,
+            }
+            ```
+
+    Returns
+    -------
+    RunSchema
+        Dictionary of results from [quacc.schemas.ase.summarize_run][]
+    """
+    # Default is phonon at gamma, with tight convergence
+    # and a lower alphamix(1) as it is very very often
+    # recommended in the QE mailing list...
+    calc_defaults = {
+        'input_data': {
+            'inputph': {
+                'tr2_ph': 1e-16,
+                'alpha_mix(1)': 0.1,
+                'nmix_ph': 12,
+                'verbosity': 'high',
+
+            }
+        },
+        'qpts': [(0, 0, 0)]
+    }
+
+    copy_files = copy_files or \
+        ['pwscf.xml',
+         'pwscf.save/data-file-schema.xml',
+         'pwscf.save/charge-density.dat.gz',
+         'paw.txt'
+         ]
```

Comment:
This will work locally, but once you start deploying calculations remotely with a workflow manager, there is no guarantee about where your current working directory will be. As such, copying from a relative file path won't work anymore.

Reply:
It looks like what you're actually trying to do is inject workflow logic into a job.

Perhaps what makes more sense is to make a custom `@flow` that consists of two `@job`s: `relax_job` and then `ph_job`. This then gives you more control over things. For instance, you'll know where the previous job's files are: it'd be `relax_job_output["dir_name"]`, assuming `relax_job_output` is the output of `relax_job`.